client/src/Components/CopyURL/CopyURL.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import LinkIcon from '@material-ui/icons/Link';
+import CopyURLStyles from './CopyURLStyles';
+import Chip from '@material-ui/core/Chip';
+import Popover from '@material-ui/core/Popover';
+import Typography from '@material-ui/core/Typography';
+
+const CopyURL = () => {
+  const classes = CopyURLStyles();
+  const url = window.location.href;
+
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleClick = event => {
+    setAnchorEl(event.currentTarget);
+    setTimeout(handleClose, 2000);
+  };
+
+  const open = Boolean(anchorEl);
```

Comment:
ja jestem zdania że takie rzutowanie typów można se zrobić w ramach jsx, ale jak chcesz na to zmienną to spoko tylko nazwij ją proszę isOpen